lib/eunit/test/eunit_SUITE.erl

```diff
@@ -203,3 +208,40 @@ check_surefire(Module) ->
 	%% Check that file is valid XML
 	xmerl_scan:file(File),
 	Chars.
+
+scale_timeouts_test(_Config) ->
+    %% Scaling with integers
+    Millis1 = run_eunit_test_that_times_out(times_out_test_,
+                                            [{scale_timeouts, 2}]),
+    ?assert(Millis1 >= 2000, #{duration => Millis1}),
+
+    %% Scaling with float: should get rounded
+    %% Scaling down should work too
+    Millis2 = run_eunit_test_that_times_out(times_out_test_,
+                                            [{scale_timeouts, 0.25}]),
+    ?assert(Millis2 >= 250, #{duration => Millis2}),
+    %% It should be possible to scale the default timeout as well
+    Millis3 = run_eunit_test_that_times_out(times_out_default_test,
+                                            [{scale_timeouts, 0.01}]),
+    ?assert(Millis3 > 0, #{duration => Millis3}),
+    ?assert(Millis3 < 5000, #{duration => Millis3}),
```

Comment:
can we be a little more aggressive in this condition as is scaling factor in this test ...

Reply:
Lowered the upper bounds check.